packages/color-scale-block/src/ColorScaleBlock.spec.ct.tsx

```diff
@@ -0,0 +1,641 @@
+        });
+
+        const COLOR_PICKER_FLYOUT_COLOR_INDEX = 0;
+
+        mount(<ColorScaleBlockWithStubs />);
+
+        cy.get(ColorScaleBlockEditorModeButtonsSelector).find(AddColorButtonSelector).contains('Add Color').realClick();
+
+        cy.get(ColorPickerFlyoutSelector)
+            .find(ColorPickerFlyoutBrandColorSelector)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .find('span')
+            .invoke('attr', 'style')
+            .as('style_1');
+
+        cy.get(ColorPickerFlyoutSelector)
+            .find(ColorPickerFlyoutBrandColorSelector)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .realClick();
+
+        cy.get(ColorPickerFlyoutSelector)
+            .parentsUntil('[role="dialog"]')
+            .find('[data-test-id="button-text"]')
+            .contains('Confirm')
+            .realClick()
+            .wait(1000);
+
+        cy.get(ColorScaleBlockSelector)
+            .find(ColorSquareBackgroundSelector)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .invoke('attr', 'style')
+            .as('style_2');
+
+        cy.get('@style_1').then((style_1) => {
+            cy.get('@style_2').then((style_2) => {
+                expect(style_2).to.include(style_1);
+            });
+        });
+    });
+
+    it('allows the user to delete a color square', () => {
+        const [ColorScaleBlockWithStubs] = withAppBridgeBlockStubs(ColorScaleBlock, {
+            editorState: true,
+            blockSettings: { customHeight: false, heightSlider: '96px', heightInput: '100px' },
+        });
+
+        const COLOR_PICKER_FLYOUT_COLOR_INDEX = 1;
+
+        mount(<ColorScaleBlockWithStubs />);
+
+        cy.get(ColorScaleBlockSelector).find(ColorSquareBackgroundSelector).should('have.length', 0);
+
+        cy.get(ColorScaleBlockEditorModeButtonsSelector).find(AddColorButtonSelector).contains('Add Color').realClick();
+
+        cy.get(ColorPickerFlyoutSelector)
+            .find(ColorPickerFlyoutBrandColorSelector)
+            .eq(COLOR_PICKER_FLYOUT_COLOR_INDEX)
+            .realClick();
+
+        cy.get(ColorPickerFlyoutSelector)
+            .parentsUntil('[role="dialog"]')
+            .find('[data-test-id="button-text"]')
+            .contains('Confirm')
+            .realClick()
+            .wait(500);
+
+        cy.get(ColorScaleBlockSelector).find(ColorSquareBackgroundSelector).should('have.length', 1);
+
+        cy.get(ColorScaleBlockSelector)
+            .find(ColorSquareWrapperSelector)
+            .eq(0)
+            .realHover()
+            .wait(500)
+            .find('[name="IconTrashBin16"]')
+            .realClick()
+            .wait(500);
+
+        cy.get(ColorScaleBlockSelector).find(ColorSquareBackgroundSelector).should('have.length', 0);
+    });
+
+    it('resizes all color square to take up equal amounts of space in the block when the user clicks on the Resize Evenly button', () => {
```

Comment:
I was able to update most of the tests so that colors are pre-populated. However, for the test that verifies if the 3rd color is 18px wide (the 2nd to last test), it requires that the pre-populated color squares have a width that is 50% of the block. Since I don't know how wide those squares should be, I am adding those squares manually within the test.

Silly question but is it possible to update the block settings after the block has been mounted?